src/Common/CustomerPortalClient.cs

```diff
@@ -401,5 +402,15 @@ public async Task<CustomerEnvironmentApplicationPackage> CreateOrUpdateAppInstal
                 CustomerLicenseName = customerLicenseName
             }.CreateOrUpdateAppInstallationAsync(true)).CustomerEnvironmentApplicationPackage;
         }
+
+
+        public async Task<CheckStartDeploymentConnectionOutput> CheckStartDeploymentConnection(CustomerEnvironment customerEnvironment, CustomerInfrastructure customerInfrastructure)
+        {
+            return (await new CheckStartDeploymentConnectionInput()
+            {
+                CustomerEnvironment = customerEnvironment,
+                CustomerInfrastructure = customerInfrastructure
+            }.CheckStartDeploymentConnectionAsync(true));

```

Comment:
The output only has one bool property so consider returning that value instead of the entire output as this is not needed. It also helps that who calls this method will no longer need to "know" the output object

Reply:
Suggestions applied. Like was talked, the async need to continue with the flag with true value, to continue the context on the powershell usage.